src/cookie.rs

```diff
@@ -292,7 +292,7 @@ where
         let reply = ListFontsWithInfoReply::try_from(reply.as_ref()).map_err(ReplyError::from);
         if reply.is_ok() {
             // Is this an indicator that no more replies follow?
-            if !reply.as_ref().unwrap().name.is_empty() {
+            if !reply.as_ref().ok().unwrap().name.is_empty() {
```

Comment:
Huh? This is `Result::ok()`, I think. Seems unnecessary to me, or am I missing something?

Reply:
Indeed, it is not needed now. I added it was at some point, but I'll remove it.

Reply:
Changed